Tests/format-HTML.Tests.ps1

```diff
@@ -0,0 +1,77 @@
+$TestsPath = Split-Path $MyInvocation.MyCommand.Path
```

Comment:
As discussed, this test is not needed anymore as we won't introduce the cmdlet that goes with it. Can you please remove it?